lib/modules/hosts/peertube.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+
+import { Host } from '../../core/host';
+
+
+export default new Host('peertube', {
+	name: 'peertube',
+	domains: [
+		'peervideo.net',
+		'peertube.social',
+		'peertube.mastodon.host',
+		'evertron.tv',
+		'mplayer.demouliere.eu',
+		'cloud.allplayer.tk',
+		'video.tedomum.net',
+		'peertube.fr',
+		'hostyour.tv',
+		'videobit.cc',
+		'videoshare.cc',
+		'peertube.openstreetmap.fr',
+		'video.ploud.fr',
+		'tube.kdy.ch',
+		'lostpod.space',
+		'pe.ertu.be',
+		'peertube.live',
+		'peer.tube',
+		'watching.cypherpunk.observer',
+		'queertube.org',
+		'exode.me',
+		'framatube.org',
+	],
+	attribution: false,
+	detect: ({ pathname, hostname }) => {
+		// split path excluding first /
+		const split = pathname.substring(1).split('/');
+		return [split[2], hostname];
```

Comment:
Are any way to check that a given link points to a video, to avoid that a link to `https://peertube.openstreetmap.fr/about` generates an expando?

Reply:
oh yes.
sending update